### ISSUE TITLE
Last word cut off on android while chatting

### ISSUE THREAD

**Issue by Ablu** [tales/tales-client#48]
On some keyboards the last typed word does not seem to be send. I had this issue somewhen ago on my tablet, but I am no longer able to reproduce it with the stock keyboards. Maybe it is related to the swift keyboard.

 frainfreeze noticed this.

**Comment by frainfreeze** [comment]
Nope, it's not the swift keyboard, issue seems to appear even when it s turned off.

**Comment by Ablu** [comment]
Yep. I was able to reproduce it on my device now too.
